py/selenium/webdriver/chromium/webdriver.py

```diff
@@ -47,7 +48,9 @@ def __init__(
          - service - Service object for handling the browser driver if you need to pass extra details
          - keep_alive - Whether to configure ChromiumRemoteConnection to use HTTP keep-alive.
         """
-        self.service = service
+
+        self.service = service if service else ChromiumService()
```

Comment:
Do we really want to instantiate the service here?

Reply:
This statement existed in code. It generated mypy errors because it can take none. 
Again took reference from WebDriver file in Firefox module to fix this. 

So what should be changed?